pkg/cluster/internal/providers/docker/provision.go

```diff
@@ -243,6 +243,11 @@ func runArgsForNode(node *config.Node, clusterIPFamily config.ClusterIPFamily, n
 		"--volume", "/var",
 		// some k8s things want to read /lib/modules
 		"--volume", "/lib/modules:/lib/modules:ro",
+		// propagate KIND_EXPERIMENTAL_CONTAINERD_SNAPSHOTTER to the entrypoint script
+		"-e", "KIND_EXPERIMENTAL_CONTAINERD_SNAPSHOTTER",
```

Comment:
Is there any reason to not just enable this when rootless and forget the env?

Reply:
This env var is for users who want to opt-in to `KIND_EXPERIMENTAL_CONTAINERD_SNAPSHOTTER=overlayfs` (kernel-mode real overlayfs) for better performance.
This env var is not needed for most users.

Reply:
Ah, hopefully in the future there some way we can check if real overlayfs is viable and automate selection.